git/filter_process_scanner.go

```diff
@@ -98,25 +98,25 @@ func (o *FilterProcessScanner) Init() error {
 // capabilities given to LFS by the parent, an error will be returned. If there
 // was an error reading or writing capabilities between the two, an error will
 // be returned.
-func (o *FilterProcessScanner) NegotiateCapabilities() error {
```

Comment:
I probably thought that `parent` was good terminology when I wrote this 😆. I removed it from this godoc in 7864e70.